packages/framework/presence/.eslintrc.cjs

```diff
@@ -19,6 +19,7 @@ module.exports = {
 		// TODO: Reenable no-explicit-any once need with ValueDirectoryOrState is
 		// understood. If `any` is still needed disable is on a per line basis.
 		"@typescript-eslint/no-explicit-any": "off",
+		"import/no-internal-modules": "off",
```

Comment:
Not sure it's necessary, but you can limit this to only permitting imports you allow. Look for the rule name in other configs for example. That is a more targeted change.

Reply:
We don't want to turn it off. We can update the rules to allow the extended internal path. (The rules are incorrect.)
```suggestion
		"import/no-internal-modules": [
			"error",
			{
				"allow": [
					"@fluidframework/*/internal**",
					"*/index.js"
				],
			},
		],
```